packages/strapi-plugin-i18n/admin/src/components/ModalEdit/index.js

```diff
@@ -31,51 +39,60 @@ const ModalEdit = ({ localeToEdit, onClose, locales }) => {
         onSubmit={handleSubmit}
         validationSchema={object().shape({
           displayName: string().max(50, 'Settings.locales.modal.edit.locales.displayName.error'),
         })}
       >
-        {({ values, handleSubmit, handleChange, errors }) => (
+        {({ handleSubmit, errors }) => (
           <form onSubmit={handleSubmit}>
-            <ModalHeader
-              headerBreadcrumbs={[formatMessage({ id: getTrad('Settings.list.actions.edit') })]}
-            />
-            <ModalSection>
-              <div>
-                <span id="locale-code">
-                  <Label htmlFor="">
-                    {formatMessage({ id: getTrad('Settings.locales.modal.edit.locales.label') })}
-                  </Label>
-                </span>
-
-                <Select
-                  aria-labelledby="locale-code"
-                  options={options}
-                  defaultValue={defaultOption}
-                  isDisabled
-                />
-              </div>
-
-              <div>
-                <Label htmlFor="displayName">
-                  {formatMessage({
-                    id: getTrad('Settings.locales.modal.edit.locales.displayName'),
-                  })}
-                </Label>
-                <InputText name="displayName" value={values.displayName} onChange={handleChange} />
+            <div className="container-fluid">
+              <div className="container-fluid">
+                <HeaderModalTitle
+                  style={{
+                    fontSize: '1.8rem',
+                    height: '65px',
+                    fontWeight: 'bold',
+                    alignItems: 'center',
+                    marginBottom: '-39px',
+                    paddingTop: '16px',
+                  }}
+                >
+                  Configurations
+                </HeaderModalTitle>
 
-                {errors.displayName && (
-                  <small>
-                    {formatMessage({
-                      id: getTrad(' Settings.locales.modal.edit.locales.displayName.error'),
+                <ModalForm>
+                  <TabsNav
+                    defaultSelection={0}
+                    label={formatMessage({
+                      id: getTrad('Settings.locales.modal.edit.tab.label'),
                     })}
-                  </small>
-                )}
+                    id="i18n-settings-tabs"
+                  >
+                    <Tabs position="right">
+                      <Tab>{formatMessage({ id: getTrad('Settings.locales.modal.base') })}</Tab>
+                      <Tab>{formatMessage({ id: getTrad('Settings.locales.modal.advanced') })}</Tab>
+                    </Tabs>
+
+                    <TabsPanel>
+                      <TabPanel>
+                        <BaseForm options={options} defaultOption={defaultOption} />
+                      </TabPanel>
+                      <TabPanel>advanced</TabPanel>
+                    </TabsPanel>
+                  </TabsNav>
```

Comment:
ℹ️ This is a suggestion to create navigable tabs. We can also add the power to navigate with the keyboard between them quite easily if necessary 😄

Reply:
I like that API 👍🏽 
What are your thoughts on the "Compound Components"
https://kentcdodds.com/blog/compound-components-with-react-hooks